src/Tickets/Flexible_Tickets/Series_Passes.php

```diff
@@ -281,8 +283,8 @@ public function unregister(): void {
 			$this,
 			'filter_tickets_attendees_report_js_config'
 		] );
-
 		remove_filter( 'tribe_template_after_include:tickets/v2/tickets/title', [ $this, 'render_series_passes_header_in_frontend_ticket_form' ], 10, 3 );
+		remove_filter( 'tribe_template_pre_html:tickets/admin-views/editor/panel/header-image', [ $this, 'hide_header_image_option_from_ticket_settings' ], 10, 5 );
```

Comment:
The way you're removing the header image on Series is correct, but the `Series_Passes` controller is not the right place to do it.

The task is to remove the Ticket header image from the Series **post**, not the Series_Passes **ticket**.
For this reason, I think this change  should be done in the `Base` controller, not this one.

Reply:
Agreed, moved it now. I started the code from the series pass perspective and hence used that controller.